README.md

```diff
@@ -107,6 +152,10 @@ get in touch with the package author if support is needed:
 
 ## Getting Started
 
+<a href="https://university.influxdata.com/catalog/">
+    <img src="assets/influxdbU-telegraf-banner.png" width="600"/>
+</a>
```

Comment:
I recommend we move this to the bottom of the `Getting Started` section otherwise it overshadows the commands that are helpful at a quick glance.